kolibri/logger/serializers.py

```diff
@@ -20,7 +20,7 @@ class ExamLogSerializer(KolibriModelSerializer):
     score = serializers.SerializerMethodField()
 
     def get_progress(self, obj):
-        return obj.exam.question_count
+        return obj.attemptlogs.values_list('item').distinct().count()
```

Comment:
So the purpose of this new implementation is de-duplicate logs that reference the same Exercise in the Exam?

For example, if I have a 3-exercise Exam, but have these logs:

* Log for Exercise 1
* Log for Exercise 1
* Log for Exercise 1
* Log for Exercise 2

Then `get_progress` will return 2 instead of 4?

Reply:
> Then get_progress will return 2 instead of 4?
Yeah, it will only return 2 examAttempt.